File: VB/CustomMapColorizer/RangeColor.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Windows.Media

Namespace CustomMapColorizer
	Public Class RangeColor
		Private ReadOnly rangeMin_Renamed As Integer
		Private ReadOnly rangeMax_Renamed As Integer
		Private ReadOnly fill_Renamed As Color

		Public ReadOnly Property RangeMin() As Integer
			Get
				Return rangeMin_Renamed
			End Get
		End Property
		Public ReadOnly Property RangeMax() As Integer
			Get
				Return rangeMax_Renamed
			End Get
		End Property
		Public ReadOnly Property Fill() As Color
			Get
				Return fill_Renamed
			End Get
		End Property

		Public Sub New(ByVal rangeMin As Integer, ByVal rangeMax As Integer, ByVal fill As Color)
			Me.rangeMin_Renamed = rangeMin
			Me.rangeMax_Renamed = rangeMax
			Me.fill_Renamed = fill
		End Sub
	End Class
End Namespace

File: CS/CustomMapColorizer/RangeColor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows.Media;

namespace CustomMapColorizer
{
    public class RangeColor
    {
        readonly int rangeMin;
        readonly int rangeMax;
        readonly Color fill;

        public int RangeMin
        {
            get { return rangeMin; }
        }
        public int RangeMax
        {
            get { return rangeMax; }
        }
        public Color Fill
        {
            get { return fill; }
        }

        public RangeColor(int rangeMin, int rangeMax, Color fill)
        {
            this.rangeMin = rangeMin;
            this.rangeMax = rangeMax;
            this.fill = fill;
        }
    }
}

